scripts/patch-realsense-ubuntu-lts-hwe.sh

```diff
@@ -330,6 +330,7 @@ fi
 # As a precausion start with unloading the core uvcvideo:
 try_unload_module uvcvideo
 try_unload_module videobuf2_v4l2
+[ ${k_maj_min} -ge 608 ] && try_unload_module videobuf2_memops
```

Comment:
For some reason, "videobuf2_memops" is not getting removed even if it is not used by any. So removing manually.

Reference output:
```
$ lsmod | grep video
videobuf2_memops       16384  0
videobuf2_common       81920  1 videobuf2_memops
mc                     81920  1 videobuf2_common
video                  73728  1 nouveau
wmi                    28672  6 video,intel_wmi_thunderbolt,gigabyte_wmi,wmi_bmof,mxm_wmi,nouveau
```